terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars

```diff
@@ -0,0 +1,15 @@
+aws_bucket = "development-eu-west-2.terraform-state.ch.gov.uk"
```

Comment:
Need to check with @tobafolaranmi-ch on some of these. We only had `environment` and `aws_profile` defined for OE web.

Reply:
Only using these two variables.